erpnext/selling/doctype/customer/customer_dashboard.py

```diff
@@ -12,7 +12,8 @@ def get_data():
 			'Payment Entry': 'party',
 			'Quotation': 'party_name',
 			'Opportunity': 'party_name',
-			'Contract': 'party_name'
+			'Contract': 'party_name',
+			'Subscription': 'party'
```

Comment:
@hrwX  Make sure subscription has party link field? change this to customer.

Reply:
@vishdha New features were merged in (subscription)[https://github.com/DigiThinkIT/erpnext/pull/589], in which customer has been removed and replaced with `party_type` and `party`.